modules/macros/pages/link-macro-attribute-parsing.adoc

```diff
@@ -22,26 +22,31 @@ The following example shows a URL macro with custom linked text alongside named
 https://discuss.asciidoctor.org[Discuss Asciidoctor,role=resource,window=_blank]
 ----
 
-If you want to define a single named attribute, but leave the linked text empty (so it falls back to the URL), then you need to start the attribute list with a comma.
-In other words, you need to leave a slot for the linked text so that the parser knows where it ends.
-Otherwise, the parser will treat the named attribute as the linked text.
+Let's consider a case when the linked text contains a comma and macro also has named attributes.
```

Comment:
would it be better english to say "let's consider a case *where* the linked text contains a comma and *the* macro also has named attributes.
?
```suggestion
Let's consider a case where the linked text contains a comma and the macro also has named attributes.
```

Reply:
Yes! I added this in.